tamr_unify_client/dataset/resource.py

```diff
@@ -434,3 +444,88 @@ def _geo_attr_names():
             "polygon",
             "multiPolygon",
         }
+
+
+class DatasetSpec:
+    """A representation of the server view of a dataset."""
+
+    def __init__(self, client, data, api_path):
+        self._data = data
+        self.client = client
+        self.api_path = api_path
+
+    @staticmethod
+    def of(dataset):
```

Comment:
nit: it is more readable (and more copy-pasteable) if the argument to `of()` is something generic, like `resource`.
```
def of(resource):
    return DatasetSpec(resource.client, deepcopy(resource._data), resource.api_path)
```